yt_idefix/misc.py

```diff
@@ -0,0 +1,72 @@
+def remove_comments(input_text):
```

Comment:
I think this all function becomes unneeded if we leverage inheritance as I suggested

Reply:
Yes sure! Let me see how to use the inheritance.

Reply:
Refactored the code to exploit inheritance.